tools/extra/fpgadiag/fpgastats.py

```diff
@@ -27,20 +27,27 @@
 
 from __future__ import print_function
 from common import exception_quit, FpgaFinder, COMMON
-from common import hexint
+from common import convert_argument_str2hex
```

Comment:
if it's from the same library, couldn't we import it with one line? (just a style thing)

Reply:
fixed